packages/webapp/src/views/common/generate-summary-list-macro.njk

```diff
@@ -35,14 +36,24 @@
 
         {# Only process rows which are flagged for display #}
         {% if item.show == true %}
+            {# Set the value HTML #}
+            {% if item.subList == true %}
+                {% set valueHtml %}
+                  {% include "common/summary-sub-list.html" %}

```

Comment:
this is added to allow for a sub list in the summary (in this case for habitat data).